linux/signal_tools.bsh

```diff
@@ -155,13 +155,13 @@ function trap_chain()
 #**
 # .. function:: exit_chain
 #
-# In case there are multiple functions being chained together for a trap, you might not want to exit until after all the traps have a chance to go. In order to accomplish this, use :func:`exit_chain`. If it trap is not part of a chain, ``exit`` is called immediately. If the trap is part of a chain, ``exit`` is called after the chain is complete
+# In case there are multiple functions being chained together for a trap, you might not want to exit until after all the traps have a chance to run. In order to accomplish this, use :func:`exit_chain`. If it trap is not part of a chain, ``exit`` is called immediately. If the trap is part of a chain, ``exit`` is called after the chain is complete
```

Comment:
```suggestion
# In case there are multiple functions being chained together for a trap, you might not want to exit until after all the traps have a chance to run. In order to accomplish this, use :func:`exit_chain`. If the trap is not part of a chain, ``exit`` is called immediately. If the trap is part of a chain, ``exit`` is called after the chain is complete
```